content/uk/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Компоненти Kubernetes
+content_type: concept
+weight: 10
+description: >
+  Kubernetes кластер складається з компонентів, що становлять собою control plane, а також групи обчислювальних машин, що називаються nodes  
+card:
+  name: concepts
+  weight: 10
+--
```

Comment:
[Netlify build failed](https://app.netlify.com/sites/kubernetes-io-vnext-staging/deploys/62fc95096fb608000814915b#L130).
> components.md:2:1": EOF looking for end YAML front matter delimiter

Maybe this fix will make the build succeed.

```suggestion
---
```